package.json

```diff
@@ -16,7 +16,9 @@
 		"type": "git"
 	},
 	"scripts": {
-		"try": "rm -rf fissa && node cli.js fissa '{\"packageName\":\"fissa\",\"name\":\"fissa\",\"description\":\"fissa\",\"mailgun\":\"fissa\",\"author\":\"fissa\"}'",
+		"try": "rm -rf fissa && node cli.js fissa '{\"packageName\":\"fissa\",\"name\":\"fissa\",\"description\":\"fissa\",\"mailgun\":\"fissa\",\"author\":\"fissa\", \"type\":\"api\"}'",
```

Comment:
zouden we deze niet gewoon weghalen dan? dat je een keuze moet maken?

Reply:
Legacy, altijd handig toch?